### ISSUE TITLE
Mandibular osteolysis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7165]
**Preferred term label:**
Mandibular osteolysis

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 614008   PMID: 21932319  2/2

**Your nano-attribution (ORCID)**
